src/App.tsx

```diff
@@ -2,16 +2,26 @@ import { Button } from './components/ui';
 import { Setting, Timer, Todos, Summary } from './components';
 import { useTimer } from './hooks';
 import { runTour } from './lib';
+import { useAtomValue } from 'jotai';
+import { themeSettingsAtom } from './lib/atom';
```

Comment:
```suggestion
import { themeSettingsAtom } from './atom';
```
can you try that

Reply:
_"Cannot find module './atom' or its corresponding type declarations"_

Reply:
in that case, keep it like that first